src/vm/eventpipeprotocolhelper.cpp

```diff
@@ -9,7 +9,9 @@
 
 #ifdef FEATURE_PERFTRACING
 
-bool EventPipeProtocolHelper::TryParseProviderConfigurations(uint8_t *&bufferCursor, uint32_t &bufferLen, CQuickArray<EventPipeProviderConfiguration> &result)
+const uint32_t IpcStreamReadBufferSize = 8192;
```

Comment:
Can't we make this a const variable inside `EventPipeProtocolHelper` rather than a global variable?